developers/weaviate/configuration/rbac/manage-users.mdx

```diff
@@ -0,0 +1,476 @@
+---
+title: Manage users
+sidebar_label: Manage users
+sidebar_position: 1
+image: og/docs/configuration.jpg
+# tags: ['rbac', 'roles', 'configuration', 'authorization']
+---
+
+import Link from '@docusaurus/Link';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.permissions.py';
+import UserPyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.users.py';
+import OidcUserPyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.oidc.users.py';
+
+:::info Added in `v1.29` and `v1.30`
+Role-based access control (RBAC) is generally available in Weaviate from version `v1.29`.  
+User management is available from version `v1.30`.
+:::
+
+In Weaviate, Role-based access control (RBAC) allows you to define roles and assign permissions to those roles. Users can then be assigned to roles and inherit the permissions associated with those roles.
+
+On this page, you will find examples of how to programmatically **manage users** and their associated roles with Weaviate client libraries.
+
+import ConfigureRbac from '/_includes/configuration/configure-rbac.mdx';
+
+<ConfigureRbac />
+
+## User management <i class="fas fa-user-edit"></i> {#user-management}
```

Comment:
This is about the DB users, right?

Wdyt about a section like "Database user operations"? Reason being that 'manage' in "Manage users" of the page also seems to include updating user permissions

Reply:
As the `list_all` command also concerns OIDC users I wouldn't necessarily name the whole section "database users". Maybe this could work:
- User management
  - List all users
  - Create a database user
  - Delete a database user
  - Rotate database user API key